client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -183,43 +145,55 @@ def get_curl(self):
-            curl += ' -H "{}: {}" '.format(header, value)
+            curl += f" -H \"{header}: {value}\""
         return curl
 
+    # TODO: Removed deprecated `always_retry_404` and `always_retry_429` params
     def predict(
         self,
-        x,
+        x: List[Any],
         compress=False,
-        max_retries=5,
-        always_retry_404=True,
-        always_retry_429=True,
-    ):
+        prediction_id: Optional[str] = None,
+        max_retries: Optional[int] = http_session_utils.DEFAULT_MAX_RETRIES,
+        always_retry_404=False,
+        always_retry_429=False,
+        retry_status: Optional[Set[int]] = http_session_utils.DEFAULT_STATUS_FORCELIST,
+        backoff_factor: Optional[float] = http_session_utils.DEFAULT_BACKOFF_FACTOR
+        ) -> Dict[str, Any]:
         """
         Makes a prediction using input `x`.
 
-        .. versionadded:: 0.13.14
-           The `always_retry_404` parameter.
-        .. versionadded:: 0.13.12
-           The `always_retry_429` parameter.
+        .. versionadded:: 0.22.0
+           The `prediction_id` parameter.
+        .. versionadded:: 0.22.0
+           The `retry_status` parameter.
+        .. versionadded:: 0.22.0
+           The `backoff_factor` parameter.
 
         Parameters
         ----------
         x : list
             A batch of inputs for the model.
         compress : bool, default False
             Whether to compress the request body.
-        max_retries : int, default 5
-            Maximum number of times to retry a request on a connection failure.
+        prediction_id: str, default None
+            A custom str to use as the ID for the prediction request.  Defaults to a randomly generated numeric id.
+        max_retries : int, default 13
+            Maximum number of times to retry a request.
         always_retry_404 : bool, default True
-            Whether to retry on 404s indefinitely. This is to accommodate model deployment warm-up.
+            Deprecated: Whether to retry on 404s indefinitely. This is to accommodate model deployment warm-up.
         always_retry_429 : bool, default True
-            Whether to retry on 429s indefinitely. This is to accommodate third-party cluster
+            Deprecated: Whether to retry on 429s indefinitely. This is to accommodate third-party cluster
             autoscalers, which may take minutes to launch new pods for servicing requests.
-
+        retry_status : set, default {404, 429}
+            Set of status codes, as integers, for which retry attempts should be made.  Overwrites default value of
+            {404, 429}.  Expand the set to include more: retry_status={404, 429, <NEW_STATUS_CODE>}
+        backoff_parameter : float, default 0.3
+            A backoff factor to apply between attempts after the second try.  Uses standard urllib3 sleep pattern:
+            `{backoff factor} * (2 ** ({number of total retries} - 1))` with a maximum sleep time between requests of
```

Comment:
As you may or may not have noticed, Sphinx renders single backticks as italics—double backticks are needed for `monospace` (which I forget all the time myself).

```suggestion
            ``{backoff factor} * (2 ** ({number of total retries} - 1))`` with a maximum sleep time between requests of
```

Reply:
Good catch.  Thank you

Reply:
Fixed